### ISSUE TITLE
how xgboost calculate the mlogloss without probability !

### ISSUE THREAD

**Issue by nasergh** [dmlc/xgboost#1857]
Hi when we use Xgboost for multiclass problem with objective="multi:softmax" and eval metric=mlogloss 
but how xgboost calculate mlogloss without probability ?
 because there is no probability in softmax output and all formula i have seen use probability for calculating logloss
 **i want to calculate mlogloss for test data but i can't find the formula in R** 
https://www.kaggle.com/wiki/MultiClassLogLoss
 thanks
